test/acceptance/features/payment/process-first-payment.feature

```diff
@@ -0,0 +1,23 @@
+@complete
+Feature: Process Incoming First Payment Request
+
+    As a payment service
+    I want process a payment request and add the correct payment quaters
```

Comment:
Typo on quarters.  Though this test doesn't reflect this description as the name implies it's to do with scheduling but the test is for setting the account codes.